utils/rpms/daos.spec

```diff
@@ -513,6 +513,7 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 %{_bindir}/rdbt
 %{_bindir}/ring_pl_map
 %{_bindir}/smd_ut
+%{_bindir}/cli_checksum_tests
```

Comment:
We should probably remove most of the unit tests from our packages.   In CI, we have a mix.  Some do this and others build just in build/ area and then add the test to ci/test_files_to_stash.txt so they get copied to unit test stages.  Otherwise, do we need to update the change log when adding files?

Reply:
It depends on where the test is going to be used.  If it's in the UT stage(s), then it should be put into the stash.  But if the test is going to be run on the functional test stages, it goes into an RPM.

I think we do run _some_ of the "unit tests" in the functional test stages.

Reply:
I updated the patch, removing unit test files I've added in the past from the packages and added them to the test_files_to_stash.txt file. I'm pretty sure they aren't run in the functional test stages.